src/Microsoft.Cci.Extensions/Microsoft.Cci.Extensions.csproj

```diff
@@ -14,7 +14,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{2179F9B5-1DBA-4563-9402-A94DE75EA9FA}</ProjectGuid>
     <DefineConstants>$(DefineConstants);COREFX</DefineConstants>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
```

Comment:
It would probably work, but since we are moving the project.json to depend on RC2 NetStandard.Library and we want to be consistent with the tool-runtime/project.json I think it is better to have this match the 1.5